### PR TITLE
feat: add code for external components

### PR DESCRIPTION
**Pull request by blackchoey**
1. Moves code from https://github.com/OfficeDev/microsoft-teams-apps-faqplus/tree/master/Source to here
2. Remove the bot and Azure Functions part

### CODE REVIEW

File: faq-plus/cofigapp/Microsoft.Teams.Apps.FAQPlusPlus.Common/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,108 @@
+﻿// <copyright file="Constants.cs" company="Microsoft">
```
Comment (adashen) [inline faq-plus/cofigapp/Microsoft.Teams.Apps.FAQPlusPlus.Common/Constants.cs:1]: Do we need file name in header?

Comment (blackchoey) [reply faq-plus/cofigapp/Microsoft.Teams.Apps.FAQPlusPlus.Common/Constants.cs]: I will update the header to align with other samples

File: faq-plus/configapp/Microsoft.Teams.Apps.FAQPlusPlus.Configuration/Web.config [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+    For a description of web.config changes see http://go.microsoft.com/fwlink/?LinkId=235367.
+
+    The following attributes can be set on the <httpRuntime> tag.
+      <system.Web>
+        <httpRuntime targetFramework="4.6.1" />
+      </system.Web>
+  -->
+  <system.web>
+    <compilation debug="true" targetFramework="4.7" />
+    <httpRuntime targetFramework="4.6" />
```
Comment (SLdragon) [inline faq-plus/configapp/Microsoft.Teams.Apps.FAQPlusPlus.Configuration/Web.config:34]: Why use different framework, compilation use 4.7 and runtime use 4.6?

Comment (blackchoey) [reply faq-plus/configapp/Microsoft.Teams.Apps.FAQPlusPlus.Configuration/Web.config:34]: I just copied the source code here without any modification. This configuration is for compatibility, which means the app can also run in .net 4.6.

**Review by wenytang-ms — APPROVED** [review]

**Pull request merged**